copter/source/docs/systemid-mode.rst

```diff
@@ -4,7 +4,11 @@
 System Identification Mode
 ==========================
 
-This mode is for advanced users and provides a means to generate mathematical models of the vehicles flight behavior for model generation. It places the vehicle into a STABILIZE like mode, and generates bursts of signals ("chirps") injected into the control loops at various points and logs the results for math analysis and model generation later.
+This mode is for advanced users and provides a means to characterize the response of a vehicle in the frequency domain.  The flight data collected from this mode can be used in several ways:
+ - generating mathematical models of the vehicles flight behavior for model generation
+ - investigating and predicting the quality of the vehicle tune
+ - calculating flying quality metrics
+It injects an input signal, called a chirp, at various points in the control loops.  The chirp is a constant amplitude oscillation that increases in frequency from a user defined minimum to maximum frequency, generally referred to as a frequency sweep.  The injection points are defined by the :ref:`SID_AXIS<SID_AXIS>` parameter and there are points in the Attitude and Position Controller.  For any injection points in the Attitude Controller, the user is required to fly the aircraft through an underlying STABILIZE like flight mode.  For any injection points in the Position Controller, the user controls are locked out and will not be able to affect the control of the vehicle.  The resulting flight data is logged for math analysis and model generation after the flight.
```

Comment:
I think CI needs a blank line before line 11 to pass....bullet lists needs a blank line after to terminate them